ui/src/app/utils/runtime-utils.tsx

```diff
@@ -128,6 +129,19 @@ const compareMachineMemory = (oldRuntime: RuntimeConfig, newRuntime: RuntimeConf
   };
 };
 
+export const compareGpu = (oldRuntime: RuntimeConfig, newRuntime: RuntimeConfig): RuntimeDiff => {
+  const oldGpuExists = !!oldRuntime.gpuConfig;
+  const newGpuExists = !!newRuntime.gpuConfig;
+  return {
+    desc: 'Change GPU config',
+    previous: oldGpuExists ? `${oldRuntime.gpuConfig.numOfGpus} ${oldRuntime.gpuConfig.gpuType} GPU` : 'No GPUs',
+    new: newGpuExists ?  `${newRuntime.gpuConfig.numOfGpus} ${newRuntime.gpuConfig.gpuType} GPU` : 'No GPUs',
+    differenceType: (!oldGpuExists && !newGpuExists) || (oldGpuExists && newGpuExists)
+    && (oldRuntime.gpuConfig.gpuType === newRuntime.gpuConfig.gpuType &&
```

Comment:
nit: please use consistent wrapping: indentation, and && at end of the line (I believe most of our code follows that pattern). This line shows both && at beginning and end